### ISSUE TITLE
Visual difference for list and values in categories

### ISSUE THREAD

**Issue by AgaFra** [Problematy/goodmap-frontend#37]
If a category has multiple possible options, is displayed (on the let panel) exactly this same way as those which should have only one option checked. 

Distinguish choices by check box and radio button could solve this problem.

**Comment by raven-wing** [comment]
Yes. But frontend does not have any knowledge which fields should be multiple choice and which should have single choice.

So we need to also change contract between frontend and backend.

Frontend should consume additional variable in category: `is_multiple_choice`, and backend should get this from db.

**Comment by raven-wing** [comment]
Also - question: does there exists case where `multiple_choice` should have OR instead of AND logic operator? If so - this also could be adressed (though its backend only)
